### PR TITLE
Unzip file

### LINKED ISSUE #651: Unzip a File
# Aim

- To unzip a zipped file using Python
# Details

- Going to be built with zipfile module , simple yet effective.

### PR DESCRIPTION
**Pull request by Amit366**
# Description

Zipped files are converted to unzipped files

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Comment by Amit366** [comment]
Screenshot (133)_LI

**Comment by Amit366** [comment]
Screenshot (134)

@kaustubhgupta this is the working model

File: Unzip file/Readme.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# <b>Unzip File</b>
+
+[![forthebadge](https://forthebadge.com/images/badges/made-with-python.svg)](https://forthebadge.com)
+
+## Unzip File Functionalities : 🚀
+
+- First upload the stocks dataset
```
Comment (kaustubhgupta) [inline Unzip file/Readme.md:7]: The description is from previous PRs. Fix this

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Code is working. Readme update:

**Comment by Amit366** [comment]
@kaustubhgupta fixed

**Review by kaustubhgupta — APPROVED** [review]

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Review by antrikshmisri — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Comment by avinashkranjan** [comment]
@Amit366 Resolve the conflict

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**